### PR TITLE
Update Twilio

### PR DESCRIPTION
**Pull request by fainpablo**
Updated Twilio 2FA methods and its documentation. Twilio supports 2FA over phone calls, sms, and the Authy app.

https://support.twilio.com/hc/en-us/articles/223136307-Enabling-two-factor-authentication-on-your-Twilio-project

### CODE REVIEW

**Comment by fainpablo** [comment]
Screen Shot 2020-02-16 at 11 37 43

File: _data/developer.yml [hunk 1]
```diff
@@ -561,7 +561,10 @@ websites:
     img: twilio.png
     tfa:
       - sms
-    doc: https://www.twilio.com/help/faq/twilio-basics/what-happens-when-i-enable-two-factor-authentication
+      - phone
+      - proprietary
+    doc: https://support.twilio.com/hc/en-us/articles/223136307-Enabling-two-factor-authentication-on-your-Twilio-project
```
Comment (phallobst) [inline _data/developer.yml]:
```suggestion
    doc: https://support.twilio.com/hc/en-us/articles/223136307
```
The shortened link is sufficient (and probably more stable).

File: _data/developer.yml [hunk 2]
```diff
@@ -561,7 +561,10 @@ websites:
...
+    exception: Software authentication requires the Authy app.
```
Comment (phallobst) [inline _data/developer.yml]:
```suggestion
```
Listing the 2FA method as "proprietary" is sufficient, we don't list the actual app in these cases.

**Review by phallobst — CHANGES_REQUESTED** [review]

File: _data/developer.yml (hunk 2 above)
Comment (fainpablo) [reply _data/developer.yml]: I guess we should remove this exception from Sendgrid then. Am I right?

Comment (phallobst) [reply _data/developer.yml]: Correct!

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**